articles/active-directory-b2c/conditional-access-user-flow.md

```diff
@@ -157,6 +157,123 @@ After you've added the Azure AD Conditional Access policy, enable conditional ac
 
 Multiple Conditional Access policies may apply to an individual user at any time. In this case, the most strict access control policy takes precedence. For example, if one policy requires multi-factor authentication (MFA), while the other blocks access, the user will be blocked.
 
+## Conditional Access Template 1: Sign-in risk-based Conditional Access
```

Comment:
Do we want to put the suggested CA policies in a new sub-tab, "Common Conditional Access policies"? 
Similar to here: https://docs.microsoft.com/en-us/azure/active-directory/conditional-access/howto-conditional-access-policy-risk